src/components/PostModal.tsx

```diff
@@ -186,7 +187,7 @@ export const PostModal = () => {
           ></Cancel>
         </AddButtonSpace>
         <Preview src="" /*投稿した写真*/></Preview>
```

Comment:
今回の変更範囲じゃないから、別のPRで対応して欲しいのだけど、
JSXのタグ内にコミット書くのはあまり自然じゃないかも。

対応するタグの上にコメントした方が良さそう。
あとこういう時は `TODO` アノテーションをつけると後で見る人がわかりやすくなるはず！
参考文献：https://qiita.com/taka-kawa/items/673716d77795c937d422

```suggestion
        </AddButtonSpace>
        
        {/* TODO: 投稿した写真をPreviewのsrc要素の中に入れる */}
        <Preview src="" />
```

Reply:
今回は一緒のPRでも問題なさそう。